packages/animations/browser/src/render/shared.ts

```diff
@@ -183,7 +183,12 @@ if (_isNode || typeof Element !== 'undefined') {
   _query = (element: any, selector: string, multi: boolean): any[] => {
     let results: any[] = [];
     if (multi) {
-      results.push(...element.querySelectorAll(selector));
+      // DO NOT REFACTOR. It will throw callstack error on "spread(...)" push for greater node-list.
```

Comment:
This should be more descriptive.

```suggestion
      // DO NOT REFACTOR TO USE SPREAD SYNTAX.  For element queries that return sufficiently large NodeList objects, using spread syntax to populate the results array causes a RangeError due to the call stack limit being reached.
```